src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java

```diff
@@ -2063,189 +2063,212 @@ public void run2() {
-                            Materials.DimensionallyTranscendentResidue.getFluid(8214L / 2),
-                            Materials.Longasssuperconductornameforuhvwire.getMolten(tier_2_quantity)},
-                    base_time / 2, 125_435_904, 12_700);
-
-            long tier_3_quantity = 144L * base_quantity * tier_up_multiplier;
-            GT_Values.RA.addPlasmaForgeRecipe(
-                    new ItemStack[]{GT_Values.NI},
-                    new FluidStack[]{
-                            Materials.Draconium.getMolten(6L * tier_3_quantity / 24),
-                            Materials.Americium.getMolten(6L * tier_3_quantity / 24),
-                            Materials.CosmicNeutronium.getMolten(7L * tier_3_quantity / 24),
-                            Materials.Tritanium.getMolten(5L * tier_3_quantity / 24),
-                            Materials.DimensionallyTranscendentExoticCatalyst.getFluid(3605L)},
-
-                    new ItemStack[]{GT_Values.NI},
-                    new FluidStack[]{
-                            Materials.DimensionallyTranscendentResidue.getFluid(3605L),
-                            Materials.Longasssuperconductornameforuhvwire.getMolten(tier_3_quantity)},
-                    base_time / 4, 250_871_808, 13_600);
-        }
-
-        // UEV Superconductors.
+            // Coils
+            int awakened_heat = 11800;
+            int infinity_heat = awakened_heat+900;
+            int hypogen_heat = infinity_heat+900;
+            int eternal_heat = hypogen_heat+900;
+
+            if (Loader.isModLoaded("Avaritia")) {
+                // Plasma forge infinity Smelting.
+
+                int base_time = 10_000;
+
+                long fuel_quantity_1 = 114_062L;
+                GT_Values.RA.addPlasmaForgeRecipe(
+                        new ItemStack[]{GT_ModHandler.getModItem("Avaritia", "Resource", 1L, 5)},
+                        new FluidStack[]{Materials.ExcitedDTRC.getFluid(114_062L)},
+
+                        new ItemStack[]{GT_Values.NI},
+                        new FluidStack[]{Materials.DimensionallyTranscendentResidue.getFluid(fuel_quantity_1 / 2), Materials.Infinity.getMolten(64L * 144L)},
+                        base_time, 204_800_000, hypogen_heat);
+
+                long fuel_quantity_2 = 51_534L;
+                GT_Values.RA.addPlasmaForgeRecipe(
+                        new ItemStack[]{GT_ModHandler.getModItem("Avaritia", "Resource", 2L, 5)},
+                        new FluidStack[]{Materials.ExcitedDTEC.getFluid(fuel_quantity_2)},
+
+                        new ItemStack[]{GT_Values.NI},
+                        new FluidStack[]{Materials.DimensionallyTranscendentResidue.getFluid(fuel_quantity_2), Materials.Infinity.getMolten(128L * 144L)},
+                        base_time/2, 409_600_000, eternal_heat);
+            }
 
-        if (Loader.isModLoaded("miscutils")) {
+            // UV Superconductor.
+
+            {
+                long base_quantity = 2043L;
```

Comment:
so why is it 2043? it is a strange number because it is 9*227, maybe it was 2304?

Reply:
Because 2043 is the closest number to 2048, below 2048 which is also a multiple of 9 (rounded down basically).

This is because there are 9 ingredients going into the recipe, otherwise there would be non-144 multiples of molten fluids going in. Other than the actual fuel material, which is precisely calculated to give a certain EU discount for each recipe.